src/tests/unit/Button.spec.js

```diff
@@ -0,0 +1,60 @@
+import { cleanup } from "@testing-library/react";
+
+import { createButtonDriver } from "../../components/Button/Button.unit.driver";
+
+describe("Button", () => {
+  let buttonsDriver;
+
+  beforeEach(() => {
+    buttonsDriver = createButtonDriver();
+  });
+
+  afterEach(cleanup);
+
+  it("by default conformation block should be hidden", () => {
+    const confirmationWrapper = buttonsDriver.given
+      .props()
+      .when.created()
+      .then.exist();
+    expect(confirmationWrapper).toBeFalsy();
+  });
+
+  it("by default, the function should be called when press delete button", () => {
+    const mockOnClick = jest.fn();
+    buttonsDriver.given
+      .text("Delete")
+      .given.click(mockOnClick)
+      .when.deleteBtnClick();
+    expect(mockOnClick).toHaveBeenCalled();
+  });
+});
```

Comment:
```suggestion
  describe('by default', () => {
    it("conformation block should be hidden", () => {
      const confirmationWrapper = buttonsDriver.given
        .props()
        .when.created()
        .then.exist();
      expect(confirmationWrapper).toBeFalsy();
    });

    it("the function should be called when press delete button", () => {
      const mockOnClick = jest.fn();
      buttonsDriver.given
        .text("Delete")
        .given.click(mockOnClick)
        .when.deleteBtnClick();
      expect(mockOnClick).toHaveBeenCalled();
    });
  });
});
```

Reply:
When confirm  = true and press confirm button. Should call onClick and hide confirmation block